### PR TITLE
first version of support for new GenomeAnnotation and Assembly objects

### PR DESCRIPTION
**Pull request by realmarcin**

### CODE REVIEW

File: src/biokbase/narrative/common/kbtypes.py [hunk 1]
```diff
@@ -738,12 +767,24 @@ class ContigSet(tls.Unicode, TypeMeta):
         class v1_1(tls.Unicode, TypeMeta):
             """Type spec for the "ContigSet" object"""
             info_text = "KBaseSearch.ContigSet-1.1"
+    class Assembly(tls.Unicode, TypeMeta):
```
Comment (mlhenderson) [inline src/biokbase/narrative/common/kbtypes.py]: KBaseSearch.Assembly is not registered.  There is a separate ContigSet and Genome defined in Search for legacy reasons having to do with the Genome and ContigSet objects defined in KBaseGenomes.  The new Assembly and GenomeAnnotation types do not need separate Search types defined.

File: src/biokbase/narrative/common/kbtypes.py [hunk 2]
```diff
@@ -738,12 +767,24 @@ class ContigSet(tls.Unicode, TypeMeta):
...
+        class v1_0(tls.Unicode, TypeMeta):
+            """Type spec for the "Assembly" object"""
+            info_text = "KBaseSearch.Assembly-1.0"
     class Genome(tls.Unicode, TypeMeta):
         """Genome type"""
         info_text = "KBaseSearch.Genome"
         class v2_0(tls.Unicode, TypeMeta):
             """Genome object holds much of the data relevant for a genome in KBase"""
             info_text = "KBaseSearch.Genome-2.0"
+    class GenomeAnnotation(tls.Unicode, TypeMeta):
```
Comment (mlhenderson) [inline src/biokbase/narrative/common/kbtypes.py]: KBaseSearch.GenomeAnnotation is not a registered type and does not need to be defined here.

Comment (realmarcin) [inline src/biokbase/narrative/common/kbtypes.py]: Thanks, I was a little overzealous there it turns out ...

So these will never be registered types?

Comment (realmarcin) [inline src/biokbase/narrative/common/kbtypes.py]: I made a new PR removing this unnecessary code for Search.

This should be ready to deploy on CI now ...

**Comment by mlhenderson** [comment]
Ok, let's merge this in and take a look on CI Narrative.

**Pull request merged**